parent/pom.xml

```diff
@@ -4820,19 +4820,11 @@
                 <version>${junit-version}</version>
             </dependency>
             <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter-api</artifactId>
-                <version>${junit-jupiter-version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter-engine</artifactId>
-                <version>${junit-jupiter-version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.junit.vintage</groupId>
-                <artifactId>junit-vintage-engine</artifactId>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
```

Comment:
I'm using the junit-bom here to keep versions aligned. Without this, I had mixed versions for junit artifacts.
Maven was taking the version prescribed by camel when it is explicitly set in the camel-parent pom, and the version prescribed by Spring Boot Bom otherwise.